### PR TITLE
Update "get-task-object" function, change the type of first param "topology" from ^TopologyContext to ^StormTopology

### PR DESCRIPTION
**Pull request by BuDongDong**
update "get-task-object" function, change the type of first param "topology" from ^TopologyContext to ^StormTopology

### CODE REVIEW

**Comment by revans2** [comment]
@BuDongDong the changes you made look good to me.  Would it be possible to get you to file a JIRA for them?  This is mostly for tracking to make it simpler to see what has gone in.

Also I am not totally sure why, but it looks like your repository for this pull was deleted or something.  The Repository is showing up as Unknown.  Without a repo name/branch I can apply a patch, but the merge may not register appropriately when I check it in.

**Comment by d2r** [comment]
@BuDongDong any update?  Should we get a new pull request for this, or can we fix the Unknown Repository Problem another way?

**Comment by BuDongDong** [comment]
@revans2 I create a new repository, and  pull this request again. see https://github.com/apache/storm/pull/433.
I have filed a JIRA, https://issues.apache.org/jira/browse/STORM-554.

**Comment by BuDongDong** [comment]
@d2r I create a new repository, and pull this request again. see #433.
I have filed a JIRA, https://issues.apache.org/jira/browse/STORM-554.
you can merge #433.

File: storm-core/src/clj/backtype/storm/stats.clj [hunk 1]
```diff
@@ -67,7 +67,7 @@
 (defrecord RollingWindowSet [updater extractor windows all-time])
 
 (defn rolling-window-set [updater merger extractor num-buckets & bucket-sizes]
-  (RollingWindowSet. updater extractor (dofor [s bucket-sizes] (rolling-window updater merger extractor s num-buckets)) nil)
+  (RollingWindowSet. updater extractor (dofor [s bucket-sizes] (rolling-window updater merger extractor num-buckets s)) nil)
```
Comment (d2r) [inline storm-core/src/clj/backtype/storm/stats.clj]: This breaks the call to `rolling-window`. Please revert.

Comment (d2r) [inline storm-core/src/clj/backtype/storm/stats.clj]: Oh, never mind, I should be looking at #433.

@BuDongDong , thanks for the update.  Would you close this pull request?

**Comment by BuDongDong** [comment]
@d2r ok, i have closed this pull request.

**Pull request closed without merging**